### PR TITLE
Updated README to add use patterns

### PR DESCRIPTION
**Pull request by racerxdl**
*Open to Discussion*

For HUNTER-237

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,3 +64,48 @@ func main() {
...
+2019-09-16T15:35:52-03:00 | I | IO    | REQ001 | MAIN > Call0 > Call1 > Call2             | test.go:38 | Doing some IO  | {"arg0":"MyArg0","huebr":"call1arg","pop":"abcde"}
+```
+
+There are some fields that are optional (like filename/line number) but it should always follow the same pattern:
+```
+DATETIME | LEVEL | OPERATION | TAG | SCOPE | [FILENAME:LINE NUMBER] | MESSAGE | LOG FIELDS
+```
+
+*   `DATETIME` => An ISO Datetime when the log is displayed
+*   `LEVEL` => The level of the log line.
```
Comment (EmanuelJr) [inline README.md]: Alguns lugares tem ponto final e outros não nessa lista

**Comment by nemo-6** [comment]
I think the current errors should be Warning and Fatal Errors should be the only errors; Why have normal errors that can be ignored? (and also warnings on top)
Having many red things that can be ignored takes away from the importance of red things in general.

**Comment by racerxdl** [comment]
@nemo-6 well that's up to the application to decide but the thing is that there are non-fatal errors. Fatal only means the application will crash / close. But for example if an argument is wrong in a HTTP request, that's logged as error since it quits the flow of the caller, but the application remains healthy.

The idea of the warn is: If something might be wrong (because of the user or the code) that might require some attention.

**Comment by nemo-6** [comment]
Receiving wrong requests also is not an error, ending the caller flow is the perfectly expected and normal behavior for the back end. That situation should only cause an error at the caller from having done the wrong call.

**Comment by racerxdl** [comment]
The log doesn't log _code errors_ but runtime errors. A wrong request is a runtime error.

**Review by nemo-6 — APPROVED** [review]
Good job Teske, I will approve your PR but we seriously need to talk about abstractions, errors, and naming Interfaces as "Instance".

File: README.md [hunk 2]
```diff
@@ -65,3 +64,83 @@ func main() {
...
+There are some fields that are optional (like filename/line number) but it should always follow the same pattern:
+```
+DATETIME | LEVEL | OPERATION | TAG | SCOPE | [FILENAME:LINE NUMBER] | MESSAGE | LOG FIELDS
+```
+
+*   `DATETIME` => An ISO Datetime when the log is displayed
+*   `LEVEL` => The level of the log line
+    *   `I` => INFO - Shows a information usually to track what's happening inside an application
+    *   `W` => WARN - Shows a warning regarding something that went in a way that might require some attention
+    *   `E` => ERROR - Shows a application error that can be expected or not
```
Comment (silviolleite) [inline README.md]: Should use `an application`.

**Review by rogefm — APPROVED** [review]

**Pull request merged**